woocommerce/woocommerce
# Add Pricing section

### All Submissions:






### Changes proposed in this Pull Request:



This PR aims to add a Pricing section and the fields `List price` and `Sale price`.

In a follow-up PR, we should replace the component ProductSectionLayout with [FormSection](https://github.com/woocommerce/woocommerce/pull/34348). 

![screenshot-clean local-2022 08 23-10_58_00](https://user-images.githubusercontent.com/1314156/186178312-1ba79970-f4e9-4204-9eff-46cb2b3d1cfb.png)

Closes # 37-gh-woocommerce/mothra-private 44-gh-woocommerce/mothra-private .

### How to test the changes in this Pull Request:

1. Go to `Products` > `Add New (MVP)` and create a product.
2. Verify that it's possible to add a `List price`, and that the price list is correctly formatted after the input loses focus.
3. Add multiple decimal points and verify that the data is being saved in the DB correctly and that the amount is being shown as it should after leaving the input.
4. The amount should stay the same after refreshing the page.
5. Verify that it's possible to add negative numbers.
6. Go to `WooCommerce` > `Settings` > `General` (URL: /wp-admin/admin.php?page=wc-settings) and change the currency options, verify that the currency reflects the changes that you have made.
7. Taxes
- If tax is included in the price, the helper text reads: Per your store settings, tax is **included** in the price.
- If tax is excluded from the price, the helper text reads: Per your store settings, tax is **not included** in the price.

### Other information:




### FOR PR REVIEWER ONLY:
